releaseNotes/AzureDevOpsPipelineExtension.md

```diff
@@ -4,9 +4,15 @@ The Azure App Configuration extension in Azure DevOps pipeline has been publishe
 
 ![sample](pictures/AzureDevOpsExtensionVersionSample.PNG)
 
+### v2.0.11 - March, 24 2020
+* Updated [homepage documentation](https://marketplace.visualstudio.com/items?itemName=AzureAppConfiguration.azure-app-configuration-task):
+  * Added `FAQ` section.
+  * Removed `Preview` tag.
```

Comment:
* put on the separate graph.
* Azure App Configuration pipeline task for Azure DevOps is now generally available.
* Added `FAQ` section to the [documentation](https://marketplace.visualstudio.com/items?itemName=AzureAppConfiguration.azure-app-configuration-task).